cataclysmbnteam/Cataclysm-BN
# Restore MBR vest and remove ESAPI vest

Restore MBR vest and remove ESAPI vest



#### Summary



#### Purpose of change
Fixes https://github.com/cataclysmbnteam/Cataclysm-BN/issues/111


#### Describe the solution
Remove unbalanced ESAPI vest and resotre old balanced MBR vest variants.

- Revert: https://github.com/CleverRaven/Cataclysm-DDA/pull/36920
- Obsolate: ESAPI vest
- Remove Kevlar MBR. We are ptetending that empty MBR is Kevlar MBR already. So buff emepty MBR vest
- emtpy MBR vest in not craftable
- Migrate ESAPI vests and plates
- Do replacing over item groups, npcs and professions.



#### Describe alternatives you've considered
None.


#### Testing
1) Start the game
2) Make sure that MBR verst not in game and ESAPI vest obsolated.


#### Additional context
None.
